### PR TITLE
gr-bokehgui: PyBOMBS receipe added

### PR DESCRIPTION
**Pull request by kartikp1995**
Kindly review the PR and let me know if I am missing something while including the recipe. I tried to make the recipe by taking existing recipes as a reference.

Thanks.

### CODE REVIEW

File: requests.lwr [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#
```
Comment (mbr0wn) [inline requests.lwr]: `python-requests` is already a recipe, and is in PyBOMBS default recipe location.

File: bokeh.lwr [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# Boston, MA 02110-1301, USA.
+#
+
+category: baseline
+depends:
+- tornado
+- numpy
+- pyyaml
+- dateutil
+- requests
```
Comment (mbr0wn) [inline bokeh.lwr]: Change to `python-requests`

**Comment by mbr0wn** [comment]
@kartikp1995 please see my comments above.

**Comment by kartikp1995** [comment]
Hi. I am sorry. I somehow missed your previous message. Thank you for the reminder.
I will do this within 36 hours!

**Comment by kartikp1995** [comment]
Hello @mbr0wn,
Updated based on your comments. Let me know if there is anything that I have missed!

Also, do you have any suggestions on how to test the PyBomb recipe before including it to the official repository?
Thanks.

**Comment by mbr0wn** [comment]
Put the recipe into one of your local recipe caches. For me, I have PyBOMBS pointed to `~/src/gr-recipes` so my local changes are reflected there even before they're pushed.

**Pull request merged**